test/unit/legacy_wrappers/bulk.test.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+const { MongoClient } = require('../../../src/index');
+const sinon = require('sinon');
+const { expect } = require('chai');
+const { BulkOperationBase } = require('mongodb/lib/bulk/common');
+
+describe('legacy_wrappers/bulk.js', () => {
+  let collection;
+  let client;
+  let stub;
+
+  beforeEach(async function () {
+    client = new MongoClient('mongodb://iLoveJs');
+    // This is the minimal topology shim needed to get bulk apis to work
+    client.topology = {
+      s: { options: {} },
+      lastHello() {
+        return null;
+      }
+    };
+    collection = client.db().collection('a');
+    stub = sinon.stub(BulkOperationBase.prototype, 'execute').returns(Promise.resolve({ ok: 1 }));
+  });
+
+  afterEach(async function () {
+    sinon.restore();
+    client.topology = null;
+    await client.close();
+  });
+
+  describe('LegacyUnorderedBulkOperation', () => {
+    describe('callbacks', () => {
+      it('should bulk.execute(callback)', done => {
```

Comment:
can we similarly improve these `it` statements to be more descriptive of what's actually being tested? clearly indicate success/failure expectations and all that jazz? (see my long comment on api.test.js)